tool/github/resolve_outdated_review_threads.sh

```
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+$(dirname $(realpath $0))/resolve_review_threads_base.sh outdated "$@"
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)